module/books.js

```diff
@@ -0,0 +1,50 @@
+let books;
+
+if (localStorage.getItem('books') === null) {
```

Comment:
- [ ] great job with the module.. for more clarity, kindly set the local storage in its own module